### DISCUSSION TITLE
Error IImageProcessingContext

### DISCUSSION THREAD

**Discussion by IParallel** [SixLabors/ImageSharp#2526] [Q&A]
i got to my project after long time and i have randoms errors like this

Cannot convert instance argument type 'SixLabors.ImageSharp.Processing.IImageProcessingContext' to 'IImageProcessingContext'

**Comment by JimBobSquarePants** [comment]
You've not really provided any useful information here. It sounds like a namespacing issue though. 

What version are you using? What IDE? Did you upgrade from a previous version?
